docs/controls/InAppNotification.md

```diff
@@ -0,0 +1,129 @@
+# In App Notification 
+
+The *In App Notification* control offers the ability to show local notifications in your application.
+
+## Syntax
+
+The control should be placed where you want your notification to be displayed in the page, generally in the root grid.
```

Comment:
I think we should tell the dev to put this control in the Bottom of the grid because If they put at the top other controls will overlay this control.